app/simformat.hs

```diff
@@ -127,3 +134,5 @@ main = do
           for_ inputsAndOutputs $ \ (file, _, output) -> do
             putStrLn' verbose $ "Reformatting " <> file
             BS.writeFile file output
+    formatStdIn regroup = do
+     getContents >>= reformat regroup . lines >>= putStr . unlines
```

Comment:
Line 138 is one space indented to the right of like 137. I'm on my phone so I can't tell which one is wrong.